docs/getting-started-guides/coreos/bare_metal_offline.md

```diff
@@ -0,0 +1,646 @@
+          command: start
+          content: |
+            [Unit]
+            Description=fleet daemon
+            Wants=fleet.socket
+            After=fleet.socket
+            [Service]
+            Environment="FLEET_ETCD_SERVERS=http://10.20.30.40:4001"
+            Environment="FLEET_METADATA=role=node"
+            ExecStart=/usr/bin/fleetd
+            Restart=always
+            RestartSec=10s
+        - name: flannel.service
+          command: start
+          content: |
+            [Unit]
+            After=network-online.target 
+            Wants=network-online.target
+            Description=flannel is an etcd backed overlay network for containers
+            [Service]
+            Type=notify
+            ExecStartPre=-/usr/bin/mkdir -p /opt/bin
+            ExecStartPre=/usr/bin/wget -N -P /opt/bin http://<pxe server ip>/flanneld
+            ExecStartPre=/usr/bin/chmod +x /opt/bin/flanneld
+            ExecStart=/opt/bin/flanneld -etcd-endpoints http://10.20.30.40:4001
+        - name: docker.service
+          command: start
+          content: |
+            [Unit]
+            After=flannel.service
+            Wants=flannel.service
+            Description=Docker Application Container Engine
+            Documentation=http://docs.docker.io
+            [Service]
+            EnvironmentFile=-/etc/default/docker
+            EnvironmentFile=/run/flannel/subnet.env
+            ExecStartPre=/bin/mount --make-rprivate /
+            ExecStart=/usr/bin/docker -d --bip=${FLANNEL_SUBNET} --mtu=${FLANNEL_MTU} -s=overlay -H fd:// ${DOCKER_EXTRA_OPTS}
+            [Install]
+            WantedBy=multi-user.target
+        - name: setup-network-environment.service
+          command: start
+          content: |
+            [Unit]
+            Description=Setup Network Environment
+            Documentation=https://github.com/kelseyhightower/setup-network-environment
+            Requires=network-online.target
+            After=network-online.target
+            [Service]
+            ExecStartPre=-/usr/bin/mkdir -p /opt/bin
+            ExecStartPre=/usr/bin/wget -N -P /opt/bin http://<pxe server ip>/setup-network-environment 
+            ExecStartPre=/usr/bin/chmod +x /opt/bin/setup-network-environment
+            ExecStart=/opt/bin/setup-network-environment
+            RemainAfterExit=yes
+            Type=oneshot
+        - name: kube-proxy.service
+          command: start
+          content: |
+            [Unit]
+            Description=Kubernetes Proxy
+            Documentation=https://github.com/GoogleCloudPlatform/kubernetes
+            Requires=setup-network-environment.service
+            After=setup-network-environment.service
+            [Service]
+            ExecStartPre=/usr/bin/wget -N -P /opt/bin http://<pxe server ip>/kube-proxy
+            ExecStartPre=/usr/bin/chmod +x /opt/bin/kube-proxy
+            ExecStart=/opt/bin/kube-proxy \
+            --etcd_servers=http://10.20.30.40:4001 \
```

Comment:
Just FYI, this config is fine for up to v0.12, but for future revs and for working at head, `--etcd_servers` doesn't work on kube-proxy or kubelet anymore.  See what the CoreOS Guides are doing at head.  No need to fix in this PR, but remember for next time you decide to upgrade.